### ISSUE TITLE
Wrong error on Squiz.ControlStructures.SwitchDeclaration break indentation

### ISSUE THREAD

**Issue by djibarian** [squizlabs/PHP_CodeSniffer#2974]
The documentation states that "*the terminating statement is indented 4 spaces from the case or default keyword*", but doesn’t seem to be the case:

```
switch(a){
  case “b”:
    foo();
    break;
}
```

generates an error saying that the `break` statement must be indented one stop from the SWITCH statement, which would be then:

```
switch(a){
  case “b”:
  foo();
  break;
}
```

So whether the rule or the docs should be fixed.

**Comment by jrfnl** [comment]
> The documentation states that "the terminating statement is indented 4 spaces from the case or default keyword", but doesn’t seem to be the case:

@djibarian Which documentation are you referring to ? There is no such phrase/documentation in the `Squiz.ControlStructures.SwitchDeclaration` sniff.
The error thrown by the sniff - `CASE keyword must be indented 4 spaces from SWITCH keyword` - and the fixes made are in line with each other, so I don't understand what you are referring to.

If you want a sniff which indents the `break` to the same level as the `case`/`default` body, use the `PSR2.ControlStructures.SwitchDeclaration` sniff instead.

**Comment by danhunsaker** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizcontrolstructuresswitchdeclaration

**Comment by gsherwood** [comment]
@danhunsaker Thanks for pointing that out. I have corrected the description over there to reflect what the sniff actually enforces.
